assets/css/styles.css

```diff
@@ -464,6 +476,7 @@ h1 {
 
 /* Style dashboard buttons */
 .dashboard-button {
+    width: 100%;
```

Comment:
Responsiveness fix. Using `width: 600px` or any other fixed value makes it unresponsive.